### ISSUE TITLE
Critical rocket + dispenser

### ISSUE THREAD

**Issue by TekuSP** [PistonMiner/TF2Classic#11]
When critical rocket hits lvl3 dispenser, game crashes for me. Happened two times for me.

**Comment by mastercoms** [comment]
Only on level 3 dispensers?

**Comment by electricman226** [comment]
I have nothing better to do at the moment, I'll look into it and see.

**Comment by electricman226** [comment]
Only a small bit of testing; but showed that level 3 dispenser isn't the only cause. I've had it happen with level 2's aswell.

The game only crashes if a player is being healed by the dispenser whilst hit or after it is hit. I am not sure if replenishing it's health without being healed by it will prevent a crash or not. (can easily be tested with a little ent_fire !picker addhealth magic.)

I am still unsure as to why this does happen.

**Comment by TekuSP** [comment]
Yeah. Just sometimes. And just critical rocket. Why not level 1 dispenser? Why level 2 and 3?
